### PR TITLE
Pull only second part of stack trace with Angelica setups

### PR DESCRIPTION
**Pull request by Elisis**
Omits the first section of the full stack trace when an Angelica-like crash is seen

### CODE REVIEW

**Comment by Elisis** [comment]
Hate testing Actions

File: entrypoint.py.bak [hunk 1]
```diff
@@ -0,0 +1,500 @@
+import io
```
Comment (eigenraven) [inline entrypoint.py.bak]: Did your text editor's backup file make it into the commit by accident? This should probably be removed.

**Review by eigenraven — COMMENTED** [review]
Looks good, minus the .bak file making it into the git tree

File: entrypoint.py.bak (hunk 1 above)
Comment (Elisis) [reply entrypoint.py.bak]: Ah, guess it did

File: entrypoint.py [hunk 2]
```diff
@@ -76,6 +76,20 @@ def main_stack_trace(self) -> list[str]:
                 ret.append(line)
         return ret
 
+    @cached_property
+    def cause_stack_trace(self) -> list[str]:
+        in_cause = False
+        mst = self.main_stack_trace
+        ret = []
+        for line in mst:
+            if line.startswith('Caused by:'):
```
Comment (mitchej123) [inline entrypoint.py]:
```suggestion
            if in_cause is False and line.startswith('Caused by:'):
```

**Review by mitchej123 — APPROVED** [review]
small nit, otherwise looks fine

**Pull request merged**